File: 프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    o = 0
    e = 0
    answer = 0
    counter = 1
    for i in num_list:
        if counter == 1:
            o += i
            counter += 1
        else:
            e += i
            counter -= 1
    
    answer = o if o >= e else e
    
    return answer

File: 프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.js
function solution(num_list) {
    let odd_sum = 0;
    let even_sum = 0;
    for(let i=1;i<=num_list.length;i++){
        if(i%2===0){
            even_sum += num_list[i-1];
        }else{
            odd_sum += num_list[i-1];
        }
    }
    return odd_sum>=even_sum ? odd_sum : even_sum;
    
}